tests/code/test_warning.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from peewee import DoesNotExist
+from app.logic.trackVolunteerHours import prereqParticipants
+from app.controllers.admin.routes import trackVolunteerHoursPage
+from app.models.eventParticipant import EventParticipant
+
+@pytest.mark.integration
+def test_prereqParticipants():
+
+    prlist = [1, 2]
+    prereqParticipant = prereqParticipants(1, prlist)

```

Comment:
this function name changed. This would have thrown an error if you ran the test suite